content/_includes/home/contributers-and-governance.html

```diff
@@ -0,0 +1,103 @@
+*   Brandi Davis Dusenbery, Seven Bridges Genomics, Inc.; [https://orcid.org/0000-0001-7811-8613](https://orcid.org/0000-0001-7811-8613)
+*   Niels Drost, Netherland eScience Center; [https://orcid.org/0000-0001-9795-7981](https://orcid.org/0000-0001-9795-7981)
+*   Scott Edmunds, GigaScience; [https://orcid.org/0000-0001-6444-1436](https://orcid.org/0000-0001-6444-1436)
+*   Geet Duggal, DNAnexus; [https://orcid.org/0000-0003-3485-359X](https://orcid.org/0000-0003-3485-359X)
+*   Rob Finn, EMBL-EBI; [https://orcid.org/0000-0001-8626-2148](https://orcid.org/0000-0001-8626-2148)
+*   Marc Fiume, DNAstack; [https://orcid.org/0000-0002-9769-375X](https://orcid.org/0000-0002-9769-375X)
+*   Jeff Gentry, Foundation Medicine; [https://orcid.org/0000-0001-5351-8442](https://orcid.org/0000-0001-5351-8442)
+*   Kaushik Ghose, Seven Bridges Genomics, Inc; [https://orcid.org/0000-0003-2933-1260](https://orcid.org/0000-0003-2933-1260)
+*   Carole Goble, The University of Manchester; [https://orcid.org/0000-0003-1219-2137](https://orcid.org/0000-0003-1219-2137)
+*   Oliver Hofmann, University of Melbourne / bcbio-nextgen; [https://orcid.org/0000-0002-7738-1513](https://orcid.org/0000-0002-7738-1513)
+*   Hervé Ménager, Institut Pasteur; [https://orcid.org/0000-0002-7552-1009](https://orcid.org/0000-0002-7552-1009)
+*   Folker Meyer, Argonne / University of Chicago; [https://orcid.org/0000-0003-1112-2284](https://orcid.org/0000-0003-1112-2284)
+*   Tom Morris, Curoverse; [https://orcid.org/0000-0003-0435-7851](https://orcid.org/0000-0003-0435-7851)
+*   Anton Nekrutenko, The Pennsylvania State University / Galaxy Project; [https://orcid.org/0000-0002-5987-8032](https://orcid.org/0000-0002-5987-8032)
+*   Brian O'Connor, University of California Santa Cruz; [https://orcid.org/0000-0002-7681-6415](https://orcid.org/0000-0002-7681-6415)
+*   Tibor Simko, CERN, [https://orcid.org/0000-0001-7202-5803](https://orcid.org/0000-0001-7202-5803)
+*   Nihar Sheth, DNAnexus; [https://orcid.org/0000-0003-4128-4364](https://orcid.org/0000-0003-4128-4364)
+*   Stian Soiland-Reyes, University of Manchester; [https://orcid.org/0000-0001-9842-9718](https://orcid.org/0000-0001-9842-9718)
+*   James Taylor, Johns Hopkins University / Galaxy Project; [https://orcid.org/0000-0001-5079-840X](https://orcid.org/0000-0001-5079-840X)
+*   Nebojša Tijanić, Seven Bridges
+*   Ward Vandewege, Curoverse Inc. / Arvados; [https://orcid.org/0000-0002-2527-6949](https://orcid.org/0000-0002-2527-6949)
+*   Alexander Wait Zaranek, Curoverse Inc. / Arvados; [https://orcid.org/0000-0002-0415-9655](https://orcid.org/0000-0002-0415-9655)
+
+<h3 id="CWL_Leadership_Team" class="section">CWL Leadership Team <a id="#CWL_Leadership_Team" href="#CWL_Leadership_Team">§</a></h3>
+
+(Alphabetical)
+
+*   Peter Amstutz, Curii / Arvados Project; [https://orcid.org/0000-0003-3566-7705](https://orcid.org/0000-0003-3566-7705)
```

Comment:
Needs synchronization with recent changes to https://github.com/common-workflow-language/cwl-website/blob/main/README.md